src/bio2bel/sources/intact.py

```diff
@@ -0,0 +1,727 @@
+# -*- coding: utf-8 -*-
+
+"""Download and convert IntAct to BEL.
+
+Run with ``python -m bio2bel.sources.intact``
+
+`IntAct <https://www.ebi.ac.uk/intact/>`_ is a interaction database with information about interacting proteins,
+their relation, and the experiments, in which these interactions were found.
+Among the interactions that are documented in IntAct are protein modifications, associations, direct interactions,
+binding interactions and cleavage reactions.
+These interactions were grouped according to their biological interpretation and mapped to the corresponding BEL
+relation. The interactions in IntAct had a higher granularity than the interactions in BioGRID. Due to the default BEL
+namespace of protein modifications :data:`pybel.language.pmod_namespace`, the post-translational protein modification
+can be identified very accurately. For example, the glycosylation of a protein can be described in BEL by
+:data:`pybel.dsl.ProteinModification('Glyco').  Although many protein modifications had corresponding terms in BEL,
```

Comment:
Problems with: :data:`pybel.dsl.ProteinModification('Glyco')

1. Missing closing back tick
2. You don't write code inside `:data:`. If you want this to be code, then use `:code:` as the prefix. Otherwise, remove `('Glyco')` and switch to be `:class:` and not `:data:`